ci-operator/config/openshift/odo/openshift-odo-master__v4.6.yaml

```diff
@@ -45,6 +45,24 @@ tests:
           cpu: "2"
           memory: 6Gi
     workflow: ipi-aws
+- artifact_dir: /tmp/artifacts
+  as: e2e-4x-psi
+  steps:
+    test:
+    - as: e2e-4x-psi-steps
+      commands: sh scripts/openshiftci-e2e-4x-psi-tests.sh
+      credentials:
+      - mount_path: /usr/local/ci-secrets/odo-rabbitmq
+        name: odo-rabbitmq
+        namespace: test-credentials
+      env:
+      - default: /usr/local/ci-secrets/odo-rabbitmq/amqpuri
+        name: ODO_RABBITMQ_AMQP_URL
+      from: oc-bin-image
```

Comment:
Why do we need this. AFAIK we already have oc binary inside the vm, no?

Reply:
I guess I missed that. I dont think it will be a danger tho. Considering 4036 is already merged, we should get this in first so i can fix the optional job change

Reply:
We can clean this out later.

Reply:
Works for me.